cabal-install/src/Distribution/Client/ProjectPlanOutput.hs

```diff
@@ -193,7 +193,8 @@ encodePlanAsJson distDirLayout elaboratedInstallPlan elaboratedSharedConfig =
               let components =
                     J.object $
                       [ comp2str c
-                        J..= ( J.object $
+                        J..= J.object
+                             (
```

Comment:
Again a weird Allman-style bracket, but still readable in an editor.

Reply:
Not sure who's Allman (Wikipedia says, his indentation style does 8 spaces for a tab, and that's about it) but an extra line  that the new version takes is unfortunate, indeed. Again, hardly critical.